src/libraries/System.Security.Cryptography.Cose/tests/CoseTestMultiSign.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Xunit;
+
+namespace System.Security.Cryptography.Cose.Tests
+{
+    public sealed class CoseTestMultiSign
```

Comment:
There are already multi-sign tests, in `CoseMultiSignMessageTests.*`.  Why do we want/need a new class with substantially the same name?